### DISCUSSION TITLE
android build doesn't  Sync  in android studio

### DISCUSSION THREAD

**Discussion by arungowda09** [axmolengine/axmol#1800] [Q&A] [answered]
- axmol version: latest clone
- devices test on:
- developing environments
   - NDK version: r23c
   - Xcode version: 14.2+
   - Visual Studio: 
      - VS version: 2022 (17.9+)
      - MSVC version: 19.39+
      - Windows SDK version: 10.0.22621.0+
   - cmake version: 
   - android studio : 2023.2.1 
Steps to Reproduce:

1. create a sample project and open **proj.android**  in android studio Gradle sync error

same project works in command Line for  Android: `axmol build -p android -a arm64` but not in android studio  can anyone help with this

<img width="1210" alt="Screenshot 2024-04-08 at 12 09 35 PM" src="https://github.com/axmolengine/axmol/assets/126662284/c059953e-e68e-4137-92e1-e2c78ed7c2df">

**Comment by halx99** [comment]
The android studio runs on macos or windows?

**Comment by arungowda09** [comment]
macos

**Comment by halx99** [comment, marked as answer]
re-run setup.ps1,  then restart android studio

**Comment by arungowda09** [comment]
build sync success and built app successfully thank you 

what was the issue ?

**Comment by halx99** [comment]
The macos launchpad not share env var `AX_ROOT` in ~/.zshrc , need use `launchctl set env`: https://github.com/axmolengine/axmol/blob/dev/setup.ps1#L241
